docs/develop/debug-office-add-ins-in-visual-studio.md

```diff
@@ -124,70 +127,87 @@ The best method for debugging an add-in in Visual Studio 2022 depends on whether
+1. Open the [personality menu](../design/task-pane-add-ins.md#personality-menu) and then choose **Attach a debugger**. This action opens the debugging tools for the webview control that Office is using to run add-ins on your Windows computer. You can set breakpoints and step through code as described in one of the following articles:
 
     - [Debug add-ins using developer tools for Internet Explorer](../testing/debug-add-ins-using-f12-tools-ie.md)
     - [Debug add-ins using developer tools for Edge Legacy](../testing/debug-add-ins-using-devtools-edge-legacy.md)
     - [Debug add-ins using developer tools in Microsoft Edge (Chromium-based)](../testing/debug-add-ins-using-devtools-edge-chromium.md)
 
 1. To make changes to your code, first stop the debugging session in Visual Studio and close the Office application. Make your changes, and start a new debugging session.
 
-## Debug an Outlook add-in project
+## Debug an add-in project in Microsoft 365 on the web
 
-This section describes how to start and debug an Outlook add-in. Debugging is done in Outlook on the web, not Outlook on Windows.
+This section describes how to start and debug an add-in in desktop Office on the web; that is, when the **Start Action** property of the add-in project is set to **Microsoft Edge**.
 
-### Start the Outlook add-in project
+### Start the add-in project on the web
 
-Start the project by choosing **Debug** > **Start Debugging** from the menu bar or press the F5 button. Visual Studio will automatically build the solution and launch the Outlook page of your Microsoft 365 tenancy.
+Start the project by choosing **Debug** > **Start Debugging** from the menu bar or press the F5 button. Visual Studio automatically builds the solution and launches the Office application host page of your Microsoft 365 tenancy.
 
-When Visual Studio builds the project it performs the following tasks.
-
-1. Prompts you for login credentials. If you're asked to sign in repeatedly or if you receive an error that you are unauthorized, then Basic Auth may be disabled for accounts on your Microsoft 365 tenant. In this case, try using a Microsoft account instead. You can also try setting the property **Use multi-factor auth** to **True** in the Outlook Web Add-in project properties pane. See [Add-in project properties](#add-in-project-properties).
+> [!NOTE]
+> When you're debugging an add-in on the web, you may get an AADSTS50011 error similar to the following: 
+>
+>    "The redirect URI `{Full absolute URL to add-in home page}` specified in the request doesn't match the redirect URIs configured for the application  ... "
+>
+> This is caused because new web applications that are deployed to SharePoint may take up to 24 hours to be available. To make your add-in debuggable immediately, take the following steps:
```

Comment:
Maybe this? To reduce redundancy
```suggestion
> This occurs because new web applications that are deployed to SharePoint may take up to 24 hours to be available. To make your add-in debuggable immediately, take the following steps:
```